config/locales/api/en_IN.yml

```diff
@@ -0,0 +1,5 @@
+# A place-holder file for en-IN locale.
+# All translations for en-IN default to :en
+# Value of `en-IN` needs to be non-empty for rails to pick it.
+en-IN:
+  place: "holder"
```

Comment:
Is the `place` key required? If not, let's remove it. 

Do we not need this empty locale file for `en-BD` etc, since the mobile app doesn't send these locales?

Reply:
We need at-least 1 key-value pair for rails to ACK the locale. Else, it doesn't include it in the I18n.backends.available_locales, and the API will begin failing again.